### ISSUE TITLE
Internal: Remove all "Container()" contructs

### ISSUE THREAD

**Issue by S-Man42** [S-Man42/GCWizard#314]
For all constructs like 

```
[
condition ? Widget() : Container()
]
```

for not visible elements just for avoiding NULL values in Widget lists;

Replace with

```
[
if (condition) Widget
]
```

**Comment by Mike-3** [comment]
most container removed

**Comment by S-Man42** [comment]
#330

**Comment by S-Man42** [comment]
Crashes in VarCoords. Please remove from 1.4.0 and insert (with fixes) in 1.5.0

**Comment by S-Man42** [comment]
It came out, that simply replacing the Container() code is not safely possible. Some strange, unexpected behaviour may occur, for example:
- Switch Behaviour on Vanity/Multiple Numbers didn't work properly anymore
- Resistor Calculator didn't do correct calculations if you switch the first value before changing ring numbers
- ...
